crates/optimism/rpc/src/sequencer.rs

```diff
@@ -74,14 +72,27 @@ impl SequencerClient {
             .await
             .inspect_err(|err| {
                 warn!(
-                    target = "rpc::eth",
+                    target: "rpc::eth",
                     %err,
-                    "Failed to forward transaction to sequencer",
+                    "{}", error_message
                 );
             })?;
```

Comment:
this we don't need if we return the `.await` result directly then the caller can do this